### PR TITLE
Ft dashboard

### PR DESCRIPTION
**Pull request by Illustre13**
#### What does this PR do?

This PR contains the design of the (1) Dashboard Page, (2) Page to add new article, (3) and finally the article page .

#### Description of Task to be completed

In This feature i have completed the following tasks:
           (1) Added the design of the Dashboard Page 
           (2) Added the design of the Page to add new article, 
           (3) and finally added the article page .

#### How should this be manually tested?

You can access the lice website of My Brand project on Netlify using this link: https://ith-mybrand.netlify.app/

#### What are the relevant pivotal tracker stories?

==> Access My Trello Board card here: https://trello.com/c/XY2xHaXE

#### Screenshots (if appropriate)
```

### CODE REVIEW

**Comment by Illustre13** [comment]
[The Admin dashboard](https://trello.com/c/XY2xHaXE/7-the-admin-dashboard)

File: add_article.html [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+      src="https://kit.fontawesome.com/aa4811ab5f.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Add Article</title>
+  </head>
+  <body>
+    <header id="menu_header">
+      <!--<div class="logo">
+                <a href="index.html"><img src="images/logo3.PNG" alt="My logo"></a>
+                </div>  -->
```
Comment (chrisostome01) [inline add_article.html]: You can remove those comments

File: admin_login_page.html [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    <script src="https://kit.fontawesome.com/aa4811ab5f.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Login - Page</title>
+</head>
+<body>
+    <div class="container">
+        <header id="menu_header" >
+            <!--<div class="logo">
+                <a href="index.html"><img src="images/logo3.PNG" alt="My logo"></a>
+                </div>  -->
```
Comment (chrisostome01) [inline admin_login_page.html]: same here, you can remove those comments

File: admin_login_page.html [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+                <div class="kemi">
```
Comment (chrisostome01) [inline admin_login_page.html]: try here to use good name for this class

File: css/dashboard.css [hunk 4]
```diff
@@ -0,0 +1,298 @@
...
+.add_article >p{
+    text-align: center;
+}
+
+
+
+/*table tr:nth-child(odd) {
+    background-color: #21e6e6;
+  }
+*/
```
Comment (chrisostome01) [inline css/dashboard.css]: You can remove this one too

**Review by chrisostome01 — CHANGES_REQUESTED** [review]
# Well done 👍👍 @Illustre13 
I left comments on some minor things about the code that are commented try to remove them and it will be good. 
Your codes look good and keep formatting your code I saw some files that were somehow hard to read.
Great job

File: add_article.html (hunk 1 above)
Comment (Illustre13) [reply add_article.html]: Done!! Removed the comments

File: admin_login_page.html (hunk 2 above)
Comment (Illustre13) [reply admin_login_page.html]: Done!! Removed the comments

File: admin_login_page.html (hunk 3 above)
Comment (Illustre13) [reply admin_login_page.html]: Have changed the class name to logo animation; also renamed the associated css file from kemi.css to logo_animation.css

File: css/dashboard.css (hunk 4 above)
Comment (Illustre13) [reply css/dashboard.css]: Done!! Removed the comments

**Review by chrisostome01 — APPROVED** [review]

**Pull request merged**